### ISSUE TITLE
Feature request: Add resource id in onResourceError event

### ISSUE THREAD

**Issue by snebel29** [ariya/phantomjs#13248]
When using the callback onResourceError the resource id is not delivered like in other callbakc (onResourceReceived, onResourceRequested, etc

i think it coul be useful when building complex scripts,

regards

**Comment by vgrinko** [comment]
It is already there, see example in one of my previous comments.
https://github.com/ariya/phantomjs/issues/13245#issuecomment-103941040

**Comment by vgrinko** [comment]
For me "onResourceError" callback reports it as:
```
{"errorCode":301,"errorString":"Protocol \"\" is unknown","id":1,"url":""}
```
Version of my phantomjs is 1.9.7

**Comment by snebel29** [comment]
Hi vgrinko,
You are right, i was using 1.9.0 where looks it wasn't implemented yet, tested on 1.9.2 with success :-)

1.9.0
{
    "errorCode": 203,
    "errorString": "Error downloading http://127.0.0.1:4567/does-not-exist.js - server replied: Not Found ",
    "url": "http://10.110.46.155:4567/does-not-exist.js"
}

1.9.2
{
    "errorCode": 203,
    "errorString": "Error downloading http://10.110.46.155:4567/does-not-exist.js - server replied: Not Found ",
    "id": 4,
    "url": "http://10.110.46.155:4567/does-not-exist.js"
}
